spec.html

```diff
@@ -5969,9 +5976,9 @@ <h1>
       </dl>
       <emu-alg>
         1. If Type(_argument_) is not Number, return *false*.
-        1. If _argument_ is *NaN*, *+&infin;*<sub>𝔽</sub>, or *-&infin;*<sub>𝔽</sub>, return *false*.
-        1. If floor(abs(ℝ(_argument_))) &ne; abs(ℝ(_argument_)), return *false*.
-        1. Return *true*.
+        1. If _argument_ is any of *NaN*, *+&infin;*<sub>𝔽</sub>, or *-&infin;*<sub>𝔽</sub>, return *false*.
+        1. If floor(abs(ℝ(_argument_))) = abs(ℝ(_argument_)), return *true*.
```

Comment:
While I am _in general_ on board with the plan to do "if true" cases before "if false" cases, this algorithm was previously basically "if x, return false. if y, return false. if z, return false. return true" and now is "if x, return false. if y, return false, if z', return true. return false". I prefer the original.